### PR TITLE
Replace button labels with icons.

### PR DESCRIPTION
**Pull request by gabi-250**
This branch replaces the button labels with icons.

### CODE REVIEW

File: openmedia/gui/controlbox.py [hunk 1]
```diff
@@ -28,24 +29,30 @@ def __init__(self):
         self.pack_start(self.button_box, False, False, 0)
 
     def _create_buttons(self):
-        self.play_button = Gtk.Button(label=u'▶')
+        self.play_button = Gtk.Button.new_from_icon_name("media-playback-start-symbolic",
```
Comment (rbartlensky) [inline openmedia/gui/controlbox.py]: This line is too long, can you try to wrap it?

File: openmedia/gui/controlbox.py [hunk 2]
```diff
@@ -28,24 +29,30 @@ def __init__(self):
...
+                                                         Gtk.IconSize.BUTTON)
         self.play_button.connect("clicked", self._play_pause)
-        self.stop_button = Gtk.Button(label=u'■')
+        self.stop_button = Gtk.Button.new_from_icon_name("media-playback-stop-symbolic",
```
Comment (rbartlensky) [inline openmedia/gui/controlbox.py]: This one and 38 are too long also.

File: openmedia/gui/controlbox.py [hunk 3]
```diff
@@ -28,24 +29,30 @@ def __init__(self):
...
+                                                         Gtk.IconSize.BUTTON)
         self.stop_button.connect("clicked", self._stop)
-        self.play_next_button = Gtk.Button(label=">>")
+        self.play_next_button = Gtk.Button.new_from_icon_name("media-skip-forward-symbolic",
```
Comment (ReapOmen) [inline openmedia/gui/controlbox.py]: Too long as well

File: openmedia/gui/tools/iconhelp.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+
+from gi.repository import Gtk
+
```
Comment (rbartlensky) [inline openmedia/gui/tools/iconhelp.py:4]: Add an extra blank line (based on pep8 styleguide)

File: openmedia/gui/tools/iconhelp.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+def get_button_image(icon_name):
+    button_image = Gtk.Image()
+    button_image.set_from_icon_name(icon_name, Gtk.IconSize.BUTTON)
+    return button_image
+
```
Comment (rbartlensky) [inline openmedia/gui/tools/iconhelp.py:18]: Remove blank line

**Review by rbartlensky — APPROVED** [review]
You can squash.

**Pull request merged**